claat/cmd/export.go

```diff
@@ -41,6 +41,8 @@ type CmdExportOptions struct {
 	GlobalGA string
 	// Output is the output directory, or "-" for stdout.
 	Output string
+	// PassMetadata are the extra metadata fields to pass along.
+	PassMetadata map[string]bool
```

Comment:
I'd make this a custom type

Reply:
map[string]bool is a common enough pattern that I think aliasing it wouldn't help readability much